modules/setting/ui.go

```diff
@@ -64,6 +64,8 @@ var UI = struct {
 		Description string
 		Keywords    string
 	} `ini:"ui.meta"`
+
+	CommitLanguage string `ini:"ui.commit_language"`
```

Comment:
if it'sjust a single string and not a subsection you dont have to do it this way

Reply:
I just tried to do this the way it was suggested in https://github.com/go-gitea/gitea/issues/26982#issuecomment-1712509924 . Could you, please, be more specific and tell me what exactly is not suitable so I could consider and change it?

Reply:
Should it be just as the following?
```go
	CommitLanguage string `ini:"COMMIT_LANGUAGE"`
```